SU-HSDO/suhumsci
# BUGHERD FIXUPS

# READY FOR REVIEW

# Summary
- Added accent dash to Featured Publications listing on person nodes to mirror mockups
- Tightened up spacing for exposed filters
- Updated secondary button styles to mirror mockup
- Un-hid yellow accent color when hovering on main menu
- Updated homepage featured publications view
- Added band-aid fix for more links when no title is present on a horizontal card (discussion to follow for future enhancement; left my comments/suggestions on ticket)
- Reduced the depth of main menu to 2
- Updated displaying of heading dashes in necessary places

# Needed By (Date)
- End of sprint (9/13)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Test to ensure that aforementioned FIXUPS have been addressed and are propagating (I can help provide examples if needed)

# Affected Projects or Products
- suhumsci (theme)
- Archaeology

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-163
- https://stanfordits.atlassian.net/browse/ARCH-347
- https://stanfordits.atlassian.net/browse/ARCH-352
- https://stanfordits.atlassian.net/browse/ARCH-359
- https://stanfordits.atlassian.net/browse/ARCH-370
- https://stanfordits.atlassian.net/browse/ARCH-377
- https://stanfordits.atlassian.net/browse/ARCH-381
- https://stanfordits.atlassian.net/browse/HSD8-266

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
